testcontainers/config.go

```diff
@@ -0,0 +1,61 @@
+package testcontainers
+
+import (
+	"context"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// Config contains the configuration for a container service.
+type Config[T testcontainers.Container] struct {
+	// ServiceKey is the key used to identify the service in the Fiber app's state.
+	ServiceKey string
+
+	// Image is the image name to use for the container.
+	Image string
+
+	// RunFn is the function to use to run the container.
+	// It's usually the Run function from the testcontainers-go module, like redis.Run or postgres.Run,
+	// although it could be the generic [testcontainers.Run] function from the testcontainers-go package.
+	RunFn func(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (T, error)
```

Comment:
Why not call this just 	`Run`?

Reply:
I used `RunFn` as a pattern I usually choose for function variables, but I'm happy to change it if it's already explicit that it's a function 😅